src/benchmarklib/benchmark_runner.cpp

```diff
@@ -454,6 +453,7 @@ void BenchmarkRunner::_create_report(std::ostream& stream) const {
                         {"summary", std::move(summary)},
                         {"table_generation", _table_generator->metrics}};
 
+  // Add information that was gathered asynchronously to the output data
```

Comment:
How and where was that gathered asynchronously?

Reply:
Grep for `INSERT INTO`. Just realized that only the system utilization is gathered asynchronously.

> Add information that was temporarily stored during the benchmark execution?

Reply:
Looking at the `INSERT INTO` part makes it clear. Thanks. In this case, the current comment is fine.